modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -35,6 +35,31 @@ def index
         render json: { data: adapted_full_list, meta: { errors: error_list } }, status: status_code
       end
 
+      def get_claim
+        claim = claims_scope.find_by(evss_id: params[:id])
+        if claim
+          raw_claim = claims_service.find_claim_by_id(claim.evss_id).body.fetch('claim', {})
+          claim.update(data: raw_claim)
+          claim_detail = EVSSClaimDetailSerializer.new(claim)
+          render json: Mobile::V0::ClaimSerializer.new(claim_detail)
+        else
+          render json: { error: "Claim #{params[:id]} not found" }, status: :not_found
```

Comment:
Could the error response match the errors schema ([json api errors format](https://jsonapi.org/examples/#error-objects))?

Reply:
If you raise a `Common::Exceptions::RecordNotFound` error it lets you [pass in an id](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/exceptions/record_not_found.rb#L9-L14) and the base controller will render the error and set the status for you.